docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,390 +1,552 @@
+The rectangular frame with a given thickness. The whole frame is plotted inside the area designated by the BorderVisual size as long as it is possible, e.g.
+- the BorderVisual of a size 200x200 and the border width of 100 will look like a square of the size of 200x200 of a solid color - the border color
+- the BorderVisual of a size 100x200 and the border width of 200 will look like a rectangle of the size of 300x200 in the border color   
 
-    The following example configures a visual transform to resize an image visual to 40 x 40 pixels and center it at the beginning of the control, with a 10 pixel horizontal offset:
 
-    ```
-    OnRelayout(Vector2 viewSize, ...)
-
-    /// Configure the image visual transform and size
-    PropertyMap imageVisualTransform = new PropertyMap();
-    imageVisualTransform.Add((int)VisualTransformPropertyType.Offset, new PropertyValue(new Vector2(10.0f, 0.0f)))
-                        .Add((int)VisualTransformPropertyType.OffsetPolicy, new PropertyValue(new Vector2((int)VisualTransformPolicyType.Absolute, (int)VisualTransformPolicyType.Absolute)))
-                        .Add((int)VisualTransformPropertyType.SizePolicy, new PropertyValue(new Vector2((int)VisualTransformPolicyType.Absolute, (int)VisualTransformPolicyType.Absolute)))
-                        .Add((int)VisualTransformPropertyType.Size, new PropertyValue(new Vector2(40.0f, 40.0f)))
-                        .Add((int)VisualTransformPropertyType.Origin, new PropertyValue((int)Visual.AlignType.CenterBegin))
-                        .Add((int)VisualTransformPropertyType.AnchorPoint, new PropertyValue((int)Visual.AlignType.CenterBegin));
-    _imageVisual.SetTransformAndSize(imageVisualTransform, size);
-    ```
+The table below lists the supported properties.
 
-<a name="bordervisual"></a>
-## Border Visual
+**Table: BorderVisual properties**
 
-The border visual renders a solid color as an internal border to the control's quad.
+| Property       | Type      | Required | Description                                                                          |
+|----------------|-----------|----------|--------------------------------------------------------------------------------------|
+| `Color`        | `Color`   | Yes      |  The color of the border.                                                            |
+| `BorderSize`   | `float`   | Yes      |  The width of the border in pixels.                                                  |
+| `AntiAliasing` | `bool`    | No       |  Specifies whether antialiasing of the border is required (default value - `false`). |
 
-**Figure: Border visual**
 
-![Border visual](media/border-visual.png)
 
-The following table lists the supported properties. The visual type is `Border` and the visual map is `BorderVisual`.
+**Usage:**
```

Comment:
I suggest te following heading here: 
#### Use BorderVisual

What is your opinion?

Reply:
BorderVisual example?
I'm ok with both, so decide please

(I will apply these conventions to all visuals)

Reply:
Global: Kindly apply 

#### BorderVisual

Reply:
I'm not sure I understood you.
I changed everywhere to ** ... Visual example: **
if I have to remove the 'example' word please let me know